libs/accounts/src/lib/accounts.mock.ts

```diff
@@ -167,3 +167,18 @@ export const amendGeneralAccountBalance = (
     }
   }
 };
+
+export const amendMarginAccountBalance = (
```

Comment:
Not a huge fan of mutating arguments passed to functions. I'd prefer to just pass an override object to accountsQuery and add create functions. Alternatively I'd keep these amend functions just scoped to the test itself. 

```
const accounts = aaccountsQuery({
  party: {
    accountsConnection: {
      edges: [
        createAccountFragment({
          balance: '100',
          market: {
            id: '1'
          }
        }),
        createAccountFragment({
          ...
        })
      ]
    }
  }
})
```